.gitignore

```diff
@@ -10,4 +10,5 @@ vendor/
 .DS_Store
 **/.DS_Store
 # Ignore .vscode settings folder
-.vscode/
\ No newline at end of file
+.vscode/
+package-lock.json
```

Comment:
Don't think we want to add this to the ignore list, though it's not a blocker to this merge.